File: openapi/code_samples/go/api_v1_memberships_organizations_{organization_id}_users_{id}/post.go
membership := &usersv1.CreateMembership{
  Roles: []*usersv1.Role{{Name: "admin"}},
  Metadata: map[string]string{
    "department": "engineering",
    "location":   "nyc-office",
  },
}
resp, err := scalekitClient.User().CreateMembership(
  ctx,
  "org_123",
  "usr_123",
  membership,
  false,
)
if err != nil {
  // handle error
  return
}
_ = resp

File: openapi/code_samples/python/api_v1_memberships_organizations_{organization_id}_users_{id}/post.py
from scalekit.v1.users.users_pb2 import CreateMembership
from scalekit.v1.commons.commons_pb2 import Role

membership = CreateMembership(
    roles=[Role(name="admin")],
    metadata={"department": "engineering", "location": "nyc-office"},
)
resp = scalekit_client.users.create_membership(
    organization_id="org_123",
    user_id="usr_123",
    membership=membership,
)